src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/data/CompareUtil.java

```diff
@@ -39,9 +39,10 @@
      * Tells if the given pattern matches the string according to the implemented comparer/algorithm.
      * @param pattern the pattern to use.
      * @param str the string to match on.
+     * @param envVars the environment variables exisiting on the jenkins host.
      * @return true if the string matches the pattern.
      */
-    boolean matches(String pattern, String str);
+    boolean matches(String pattern, String str, EnvVars envVars);
```

Comment:
Yes, sorry. The "darn it" was on me not on you :)
This plugin has been around for quite a long time so it's really hard to tell what public methods can be changed and not, not just other plugins but whatever groovy scripts that admins has laying around etc. So we try to keep binary compatibility as much as possible and only make breaking changes when absolutely nessesary.